faucet.js

```diff
@@ -154,6 +154,33 @@ class Faucet {
             collateral_amount_accumulated : res[1],
         }
     }
+
+    /**
+     * @dev send tx to cost sponsored gas fee
+     * @param privateKey begin with 'Ox'
+     * @param address test contract address
+     * @param count how many transactions, eg. 1000 
+     * @param upperBound unit CFX, eg. 0.001
+     */
+    async gasCosumeForTest(privateKey, address, count, upperBound) {
```

Comment:
Basically, this should be an external tool to help on test. You'd better revert this commit.

Reply:
removed